source/Calamari/Kubernetes/Commands/HelmUpgradeCommand.cs

```diff
@@ -71,7 +71,10 @@ public override int Execute(string[] commandLineArguments)
                 new DelegateInstallConvention(d => extractPackage.ExtractToStagingDirectory(pathToPackage)),
                 new StageScriptPackagesConvention(null, fileSystem, new CombinedPackageExtractor(log), true),
                 new ConfiguredScriptConvention(new PreDeployConfiguredScriptBehaviour(log, fileSystem, scriptEngine, commandLineRunner)),
-                new DelegateInstallConvention(d => substituteInFiles.Substitute(d, FileTargetFactory().ToList())),
+                // Any values.yaml files in any packages referenced by the step will automatically have variable substitution applied (we won't log a warning if these aren't present)
+                new DelegateInstallConvention(d => substituteInFiles.Substitute(d, DefaultValuesFiles().ToList(), false)),
```

Comment:
Is this correct? DefaultValuesFiles().ToList() would generate the same list of files each time, so the two calls to `Substitute` do the same thing (except for the last param being true instead of false)?

Reply:
Oops. No.